numba/byteflow.py

```diff
@@ -587,8 +587,14 @@ def op_YIELD_VALUE(self, state, inst):
         state.push(res)
 
     def op_RAISE_VARARGS(self, state, inst):
+        in_exc_block = state.get_top_block("EXCEPT") is not None
         if inst.arg == 0:
             exc = None
+            if in_exc_block:
+                raise UnsupportedError(
+                    "Re-raise exception not supported, yet.",
```

Comment:
How about "The re-raising of an exception is not yet supported" ?

Reply:
applied